### ISSUE TITLE
PHPStorm 2019.1 reports Grav Support plugin as incompatible

### ISSUE THREAD

**Issue by pauloffb** [PioBeat/GravSupport#7]
PHPStorm 2019.1 upgrade shows message:

> Plugin incompatible with the new build found: Grav Support

Didn't upgrade yet so don't know if it really breaks anythig...

**Comment by PioBeat** [comment]
Hello!

Yes that is correct. Currently, the plugin is configured in a way that it only "works" with non-breaking API changes of IntelliJ (i.e., 2018.2.* - 2018.3.*). This is reflected in the `plugin.xml` of the project by `<idea-version since-build="182" until-build="183.*"/>`.
This necessarely does not mean that the plugin will not work. IntelliJ will just ignore it on start-up.

I only release the plugin for versions of IntelliJ that I actually tested. By this, I can be sure that even if I do not work for a long time on the project, others can still update their IntelliJ (non-breaking changes) and the plugin should still work correctly.

At the moment, I have added some new functions and fixed some bugs. I will try do push a new release the next days and incorporate these into the new release.
If you really need the plugin, try to checkout the repository and change the `plugin.xml` and build the `*.jar` yourself. Maybe, everything will work as expected.

I hope this helps for the moment.
